TRENZ.Lib.RazorMail.Core/Models/SmtpAccount.cs

```diff
@@ -1,5 +1,37 @@
 ﻿namespace TRENZ.Lib.RazorMail.Models;
 
-public record SmtpAccount(string Host, int Port,
-                          bool TLS,
-                          string Login, string Password);
+/// <summary>
+/// Represents an SMTP account.
+/// </summary>
+public record SmtpAccount
+{
+    /// <summary>
+    /// The configuration section name.
+    /// </summary>
+    public const string SectionName = "SmtpAccount";

```

Comment:
maybe a bit tightly coupled

Reply:
It's part of the Options pattern. What do you propose?

Reply:
I guess that's fine.

Maybe move it to `Models/Configuration/` to make that clearer?

(Some of the folder structure isn't very intuitive, unrelated to this PR.)

Reply:
Sure